doc/src/message_access.md

```diff
@@ -0,0 +1,127 @@
+# Contract Messages with Special Access Needs
+Only those contracts that has at least one execute message with restricted access are included in this part.
+
+## Router
+
+### Governance-Only Messages
```

Comment:
remove `Messages` from all these section headers, mostly because `Admin Or Governance Messages` sounds a bit strange (they are not admin messages or governance messages, but messages that can only be called by either of these addresses)